client/src/app/App.tsx

```diff
@@ -286,7 +286,7 @@ class App extends Component<RouteComponentProps<AppRouteProps> & { config: IConf
                                                 )}
                                             />
                                             <Route
-                                                path="/:network/nft/:tokenId"
+                                                path="/:network/nft/:nftId"
                                                 component={(props: RouteComponentProps<NFTDetailsRouteProps>) =>
                                                 (
                                                     <NFTDetails
```

Comment:
Please add a check here to this works only if network `isStardust` (look above). If not dont render anything. Actually add it around the route so the route will exist only if currently on stardust network.

Reply:
done

Reply:
done